apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -95,6 +95,13 @@ export default class ScriptEditor extends React.Component {
       .removeAttr('selected', true);
   };
 
+  handleUmbrellaSelectChange = event => {
+    const csf = event.target.value === 'CSF';
+    this.setState({
+      csf: csf
+    });
+  };
```

Comment:
Instead of adding a `csf` attribute to state, I'd suggest putting `curriculumUmbrella` in state and making the relevant dropdown a controlled component.  I believe it removes the duplication of the `curriculumUmbrella === 'CSF'` check and may make it easier to support CSD- or CSP-specific features in the future (not that we want to).

```jsx
constructor(props) {
  super(props);
  this.state = {
    // Set default from props in the constructor:
    curriculumUmbrella: this.props.curriculumUmbrella
  };
}

// Simpler change handler
handleUmbrellaSelectChange = event => {
  const curriculumUmbrella = event.target.value;
  this.setState({curriculumUmbrella});
};

// Controlled component in render()
<select
  style={styles.dropdown}
  value={this.state.curriculumUmbrella}
  onChange={this.handleUmbrellaSelectChange}
/>

// We only check against 'CSF' here
<ProgressLegend excludeCsfColumn={this.state.curriculumUmbrella !== 'CSF'} />
```

Reply:
+1 for this!